### ISSUE TITLE
Add Notifications To Planned Payments

### ISSUE THREAD

**Issue by ivywallet** [Ivy-Apps/ivy-wallet#232]
To allow remind users to pay and not miss datelines.

**Comment by StarHamster** [comment]
Depends on #76

**Comment by Aditya137x** [comment]
Duplicate of https://github.com/Ivy-Apps/ivy-wallet/issues/76
